static/app/views/issueDetails/streamline/useSuspectFlags.tsx

```diff
@@ -0,0 +1,116 @@
+import moment from 'moment-timezone';
+
+import type {Event} from 'sentry/types/event';
+import type {Organization} from 'sentry/types/organization';
+import {trackAnalytics} from 'sentry/utils/analytics';
+import {useApiQuery} from 'sentry/utils/queryClient';
+
+type RawFlag = {
+  action: string;
+  created_at: string;
+  created_by: string;
+  created_by_type: string;
```

Comment:
make sure your backend team is camel casing responses. we do miss a lot of these though

Reply:
cc @cmanallen

Reply:
@scttcper Is there a style guide or is this convention?

Reply:
number 10 says something about it, but i've never seen this page before right now
https://develop.sentry.dev/application/api/checklist/

Reply:
@cmanallen yup, i'll do the backend changes in a separate pr